connectors/src/lib/models.ts

```diff
@@ -562,3 +562,61 @@ GoogleDriveSyncToken.init(
   }
 );
 Connector.hasOne(GoogleDriveSyncToken);
+
+export class GoogleDriveWebhook extends Model<
+  InferAttributes<GoogleDriveWebhook>,
+  InferCreationAttributes<GoogleDriveWebhook>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+  declare webhookId: string;
+  declare expiresAt: Date;
+  declare url: string;
+  declare connectorId: ForeignKey<Connector["id"]>;
+}
+
+GoogleDriveWebhook.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    connectorId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    webhookId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    expiresAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+    url: {
```

Comment:
what is URL here?

Reply:
This is our connectors URL I believe. Why store it?

Reply:
I thought it might change over time and we might want to store it. It helped me debug. I can remove it if you want. LMK.

Reply:
Let's not store N times the same value that we can reconstruct from code :+1:

Reply:
and that is not usable since it was redacted anyway right?

Reply:
> and that is not usable since it was redacted anyway right?
yes - It was mostly useful for debugging - I will remove it.